modules/crm/includes/functions-customer.php

```diff
@@ -1008,6 +1011,30 @@ function erp_crm_customer_schedule_notification() {
     }
 }
 
+/**
+ * Get current datetime of contact according to timezone
+ *
+ * @since WP_ERP_SINCE
+ *
+ * @param array $extra An associative array with a key 'client_time_zone' containing the timezone value
+ *
+ * @return string
+ */
+function erp_crm_get_current_datetime( $extra ) {
```

Comment:
It could be something like below:
```
    function erp_crm_get_current_datetime( $time_zone = false ) {
        $current_time = current_time( 'mysql' ); // the old approach: if client time zone information is not present then using site timezone
        if ( false === $time_zone ) {
            return $current_time;
        }

        try {
            $current_time = new \DateTime( 'now', new \DateTimeZone( $time_zone ) );
            $current_time = $current_time->format( 'Y-m-d H:i:s' );
        } catch ( \Exception $e ) {
        }

        return $current_time;
    }
```